commands/addons/attach.js

```diff
@@ -43,7 +43,7 @@ function * run (context, heroku) {
 module.exports = {
   topic: 'addons',
   command: 'attach',
-  description: 'attach add-on resource to a new app',
+  description: 'attach an existing add-on resource to an app',
```

Comment:
Fair. Figured I'd mention it. I don't know how confusing it really is to users, so it's probably low priority.